.github/workflows/backport.yml

```diff
@@ -0,0 +1,20 @@
+name: Backport
+on:
+  pull_request:
+    types: [closed]
+jobs:
+  backport:
+    name: Create backport PRs
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          # required to find all branches
+          fetch-depth: 0
+      - name: Create backport PRs
+        uses: zeebe-io/backport-action@9b8949dcd4295d364b0939f07d0c7593598d26cd
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_workspace: ${{ github.workspace }}
+          version: 9b8949dcd4295d364b0939f07d0c7593598d26cd
```

Comment:
Probably worth adding a comment that this needs to be in sync with `uses: zeebe-io/backport-action@***`

Reply:
Could the commit hash be exported into a variable?

Reply:
Not easily, I've added comments for now.